src/integrationTest/resources/roles.yml

```diff
@@ -17,3 +17,21 @@ user_limited-user__limited-role:
       allowed_actions:
         - "indices:data/read/get"
         - "indices:data/read/search"
+flow_framework_full_access:
+  cluster_permissions:
+    - 'cluster:admin/opensearch/flow_framework/*'
+    - 'cluster_monitor'
+  index_permissions:
+    - index_patterns:
+        - '*'
+      allowed_actions:
+        - 'indices:admin/aliases/get'
+        - 'indices:admin/mappings/get'
+        - 'indices_monitor'
+flow_framework_read_access:
```

Comment:
How come we are are modifying this role here? I am not sure how it connects to the rest of the changes outside of being used in the test cases above.

Reply:
The roles.yml used in this test case are effectively the 'defaults' - in this case I added these two roles they are part of the distribution defaults - without the reserved keyword so they could be altered using conventional REST API json content.